docs/uploader/aws-s3-multipart.mdx

```diff
@@ -390,4 +390,20 @@ or between `500` and `600`.
 
 </details>
 
+#### `shouldUseMultipart(file, fileSize)`
```

Comment:
I believe `fileSize` was removed, right?

Reply:
I saw that after the merge and updated the docs in a follow up pr, thanks.